.github/workflows/linux-qe.yml

```diff
@@ -0,0 +1,150 @@
+name: tester-linux
+
+on:
+  workflow_run:
+    workflows: [Build Windows artifacts]
+    types:
+      - completed
+
+jobs:
+  tests:
+    runs-on: [self-hosted, linux, testing-farm]
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ['1.21']
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go }}
+      
+      - name: prepare env
+        run: |
+          sudo yum install podman openssh-server git make -y
+          echo "commit_sha=${commit_sha}" >> "$GITHUB_ENV"
+          mkdir ${{ env.commit_sha }}
+          cd ${{ env.commit_sha }}
+
+          #timestamp=`date +%s`
+          #mkdir $timestamp
+          #echo "timestamp=$timestamp" >> $GITHUB_ENV
+          $cd $timestamp
+
+          ssh-keygen -t rsa -N '' -f id_rsa -q 
+          eval $(ssh-agent -s)
+          echo $SSH_AUTH_SOCK > ssh_auth_sock
+          echo $SSH_AGENT_PID > ssh_agent_pid
+          #kill $(cat ssh_agent_pid)
+          ssh-add id_rsa
+
```

Comment:
We will need api invoke to add this as check for the PR https://github.com/crc-org/crc/blob/0342835858fab4842706eb7caafb9f34bccff3f9/.github/workflows/windows-qe-tpl.yml#L84 this previously need to correlate the execution https://github.com/crc-org/crc/blob/main/.github/workflows/windows-qe-tpl.yml#L61